app/lib/actions/katello/host/update.rb

```diff
@@ -12,11 +12,11 @@ def plan(host, consumer_params = nil)
             host.content_facet.save! if host.content_facet
 
             if host.subscription_facet
-              if consumer_params
-                host.subscription_facet.update_from_consumer_attributes(consumer_params)
-              else
+              unless consumer_params
                 consumer_params = host.subscription_facet.consumer_attributes
               end
+
+              host.subscription_facet.update_from_consumer_attributes(consumer_params)
```

Comment:
whats the purpose of this change?  Seems like its needlessly updating the subscription_facet here.